app/models/email_notification.rb

```diff
@@ -0,0 +1,5 @@
+class EmailNotification < Notification
```

Comment:
Namespace your model under <code>Notification</code>

So this would be <code>Notification::Email</code>